Mathlib/SetTheory/ZFC/Basic.lean

```diff
@@ -887,8 +891,7 @@ prefix:110 "⋃₀ " => ZFSet.sUnion
 
 /-- The intersection operator, the collection of elements in all of the elements of a ZFC set. We
 special-case `⋂₀ ∅ = ∅`. -/
```

Comment:
I don't agree with "special case" here; maybe "we define" or similar instead.

Reply:
I've reworded using "define".